### PR TITLE
Add some docs on existing unit tests

### PR DESCRIPTION
**Pull request by sydhds**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

File: massa-async-pool/src/pool.rs [hunk 1]
```diff
@@ -1035,3 +1035,374 @@ impl AsyncPool {
...
+        db: Arc<RwLock<Box<dyn MassaDBController>>>,
+        column: &str,
+    ) -> BTreeMap<Vec<u8>, Vec<u8>> {
+        db.read()
+            .iterator_cf(column, MassaIteratorMode::Start)
+            // .collect::<BTreeMap<Vec<u8>, Vec<u8>>>()
+            .collect()
+    }
+
+    fn get_message() -> AsyncMessage {
```
Comment (AurelienFT) [inline massa-async-pool/src/pool.rs:1069]: I don't think this function really `get` a message but more `create` it.

Comment (Leo-Besancon) [inline massa-async-pool/src/pool.rs:1069]: Small code duplication here, no? Don't change it if it's the only duplicated helper function though.

**Review by Leo-Besancon — APPROVED** [review]
LGTM

**Comment by sydhds** [comment]
bors merge

**Review by Leo-Besancon — APPROVED** [review]

**Comment by sydhds** [comment]
bors merge

**Pull request merged**